src/components/Borrow/WrappedBorrow/WrappedBorrow.tsx

```diff
@@ -1,19 +1,32 @@
-import { printBN } from '@consts/utils'
+import { printBN, printBNtoBN } from '@consts/utils'
 import { Grid } from '@material-ui/core'
 import { ExchangeCollateralTokens, ExchangeSyntheticTokens } from '@selectors/solanaWallet'
 import { PublicKey } from '@solana/web3.js'
-import { Vault } from '@synthetify/sdk/lib/exchange'
+import { Decimal, Vault, VaultEntry } from '@synthetify/sdk/lib/exchange'
+import BN from 'bn.js'
 import React from 'react'
 import { BorrowInfo } from '../BorrowInfo/BorrowInfo'
 import { BorrowTable } from '../BorrowTable/BorrowTable'
+import {
+  calculateAvailableBorrow,
+  calculateAvailableWithdraw,
+  calculateLiqPrice
+} from '../borrowUtils'
 import { ActionBorrow } from '../SwitchBorrow/ActionBorrow'
 import ActionMenuBorrow, { IActionContents } from '../SwitchBorrow/ActionMenuBorrow'
 import useStyles from './style'
 export interface BorrowedPair extends Vault {
   collateralData: ExchangeCollateralTokens
   syntheticData: ExchangeSyntheticTokens
 }
-export interface OwnedVaults {
+interface AssetPriceData {
```

Comment:
i think this is the same type as the one used in swapline; maybe move it to file with consts?